src/ConfigCatClient.php

```diff
@@ -217,7 +231,7 @@ public function getValueDetails(string $key, mixed $defaultValue, ?User $user =
                     'event_id' => 1001,
                     'KEY' => $key,
                     'DEFAULT_PARAM_NAME' => '$defaultValue', 'DEFAULT_PARAM_VALUE' => Utils::getStringRepresentation($defaultValue),
-                    'AVAILABLE_KEYS' => implode(", ", array_keys($settingsResult->settings))
+                    'AVAILABLE_KEYS' => "'".implode("', '", array_keys($settingsResult->settings))."'"
```

Comment:
Although this would work when key count > 0 but it should result in an empty string otherwise.